app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/checkemail/CheckEmailScreen.kt

```diff
@@ -152,18 +150,13 @@ private fun CheckEmailContent(
         )
         Spacer(modifier = Modifier.height(8.dp))
 
-        val descriptionAnnotatedString = createAnnotatedString(
-            mainString = stringResource(
-                id = R.string.we_sent_an_email_to,
-                email,
-            ),
-            highlights = listOf(email),
-            highlightStyle = SpanStyle(
+        val descriptionAnnotatedString = R.string.we_sent_an_email_to.toAnnotatedString(
+            email,
```

Comment:
🔨  do we need internal param names here?

Reply:
weird `vararg` thing. if I put `args =` I need to pass in as `arrayOf(email)`